examples/vue3-ssr-vite/server.js

```diff
@@ -0,0 +1,104 @@
+// @ts-check
```

Comment:
This whole file is from [here](https://github.com/vitejs/vite/tree/b1598cec7ee185f796d8679f0a97d36b80fe1949/packages/playground/ssr-vue).